examples/plot_mne_example.py

```diff
@@ -0,0 +1,223 @@
+"""
+Using fooof with MNE
+=================
+
+This examples illustrates how to use fooof with `MNE
```

Comment:
This first line of the header gets used as the 'hover over' text - but it doesn't play nice with having links in it. 

Make this first line not have a link, and then in a subsequent line, mention briefly what MNE is, with the link to it. I updated the NDSP MNE example recently to do this, if you want to throw an eye on that.